NetKVM/Common/ParaNdis_Common.cpp

```diff
@@ -1899,8 +1899,8 @@ static VOID SetAllVlanFilters(PARANDIS_ADAPTER *pContext, BOOLEAN bOn)
 /*
     possible values of filter set (pContext->ulCurrentVlansFilterSet):
     0 - all disabled
-    1..4095 - one selected enabled
-    4096 - all enabled
+    1..4094 - one selected enabled
+    4095 - all enabled
```

Comment:
But now 4095 will not be used, right? Or in case of 0, we will set to 4095 the filter value?

Reply:
> But now 4095 will not be used, right? 

It will not be used as VlanId, but used as filter wildcard (instead of 4096)

> Or in case of 0, we will set to 4095 the filter value?

Yes

Before the patch:
4095 is valid VlanId, 0 sets filter to 4096 (wildcard value)
After:
4095 is not valid VlanId, 0 sets filter to 4095 (wildcard value)